docs/UserGuide.md

```diff
@@ -174,6 +174,20 @@ Welcome to TutorAid's beginner's tutorial! Here, you will learn how you can use
 If this is your first time launching TutorAid, you will be able to see that some sample data has been provided to you. For this tutorial, let's assume that you tutor the student(s) 
 shown in the Student Panel and that you offer the lesson(s) shown in the Lesson Panel. 
 
+### Step 2: Adding a lesson
+{:.no_toc}
+Now that you've added your first student (and perhaps a few others too), let's add some Lessons to TutorAid. In this tutorial, we will be adding a lesson named Maths 1 into TutorAid by performing the steps below:
```

Comment:
May have to update `Maths 1` to some other lesson name, since it exists in the sample data and adding of duplicate lessons is not allowed

Reply:
I agree with Jun Wei.